js/app/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "pymedphys",
+  "author": {
+    "name": "PyMedPhys",
+    "email": "developers@pymedphys.com",
+    "url": "https://pymedphys.com"
+  },
+  "description": "A community effort to develop an open standard library for Medical Physics in Python. We build high quality, transparent software together via peer review and open source distribution. Open code is better science.",
+  "licence": "Apache-2.0",
+  "version": "0.38.0",
```

Comment:
Is the idea that only a release will get built as a binary?  No "nightly"?

Reply:
Right now I have set it up to build every commit (which uses a lot of CI minutes :/). This version number here at the moment is just hard coded. I have put a todo note within the build code to rectify that:

https://github.com/pymedphys/pymedphys/blob/42e1b10a2afebefcece30640eabc13e83c2adc68/lib/pymedphys/_dev/build.py#L34

For now, it really should have something like a dev version + an appended commit hash to make that clear...